lib/std/zig/system.zig

```diff
@@ -805,6 +805,7 @@ fn glibcVerFromRPath(rpath: []const u8) !std.SemanticVersion {
         error.InvalidWtf8 => unreachable, // Windows only
         error.BadPathName => unreachable, // Windows only
         error.PipeBusy => unreachable, // Windows-only
+        error.NoDevice => unreachable, // Windows-only
```

Comment:
This isn't accurate, the `error.NoDevice` case should be moved down to the `=> return error.GLibCNotFound` branch